apf/jackclient.h

```diff
@@ -362,6 +369,10 @@ class JackClient
       (void)code;  // avoid "unused parameter" warning
       throw jack_error("JACK shutdown! Reason: " + std::string(reason));
     }
```

Comment:
I don't think this will be called. I implemented a function overload for no arguments which should be the one getting registered. However, to avoid confusion I renamed according to shutdown_info (with arguments) and shutdown (without arguments). The whole chain of events is quite convoluted but now seems to be `jack_on_shutdown` -> `_jack_shutdown_callback` -> `(cast)->jack_shutdown_callback` -> `jack_shutdown_callback`.

Reply:
> I don't think this will be called.

Yes, that's exactly my point.
It's never called, we don't need it, we should remove it. Right?

There is no need to define a `jack_info_shutdown_callback()` if we don't intend to ever use it.

Reply:
I was hoping that the windows specific issue will be resolved soon, as it seems this is related to the Jack library. 
But I understand it's unnecessary for now. Would you prefer to remove it?

Reply:
yeah, if we don't use it, we should remove it.

If the Windows issue will be solved at some point, that would be great, then we can simply change it back.